code/week6/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <opencv2/opencv.hpp>
```

Comment:
No, we'll be using CMake for all of the hardware application demos for every week. They'll be familiar with this workflow by this point in the semester.